--- devices/zwave/agozwave.cpp	2013-08-25 12:54:23.099555468 -0400
+++ devices/zwave/agozwave.cpp.patched	2013-08-28 22:58:09.907446938 -0400
@@ -674,7 +674,7 @@
 	printf("connection to agocontrol established\n");
 
 	// init open zwave
-	Options::Create( "/etc/openzwave/config/", "/etc/opt/agocontrol/ozw/", "" );
+	Options::Create( "/usr/share/openzwave/config/", "/etc/opt/agocontrol/ozw/", "" );
 	Options::Get()->AddOptionBool("PerformReturnRoutes", false );
 	Options::Get()->AddOptionBool("ConsoleOutput", false ); 
 
