--- devices/zwave/agozwave.cpp	2014-02-26 22:00:44.688349947 -0500
+++ devices/zwave/agozwave.cpp.patched	2014-02-26 22:04:59.913895248 -0500
@@ -909,7 +909,7 @@
 	printf("connection to agocontrol established\n");
 
 	// init open zwave
-	Options::Create( "/etc/openzwave/config/", CONFDIR "/ozw/", "" );
+	Options::Create( "/usr/share/openzwave/config/", CONFDIR "/ozw/", "" );
 	Options::Get()->AddOptionBool("PerformReturnRoutes", false );
 	Options::Get()->AddOptionBool("ConsoleOutput", false ); 
 
