--- bindings/qpid/perl/CMakeLists.txt.orig	2013-11-04 23:48:40.504174393 -0500
+++ bindings/qpid/perl/CMakeLists.txt	2013-11-04 23:48:46.525227802 -0500
@@ -51,7 +51,6 @@
 endif ((${CMAKE_MAJOR_VERSION} EQUAL 2) AND (${CMAKE_MINOR_VERSION} LESS 8))
 
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cqpid_perl.pm
-              ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE
               ${CMAKE_CURRENT_SOURCE_DIR}/Makefile.PL
         DESTINATION ${PERL_PFX_ARCHLIB}
         COMPONENT ${QPID_COMPONENT_CLIENT}
